### PR TITLE
[ui] make url to swagger configurable

### PR DESCRIPTION
**Pull request by xavier-rbz**
Signed-off-by: Xavier <xavier@reblaze.com>

### CODE REVIEW

File: curiefense/ui/src/components/__tests__/SideMenu.spec.ts [hunk 1]
```diff
@@ -11,16 +11,19 @@ describe('SideMenu.vue', () => {
...
       path: '/config',
     }
     kibanaURL = 'https://10.0.0.1:5601/app/discover/'
     grafanaURL = 'https://10.0.0.1:30300/'
+    swaggerURL = 'https://10.0.0.1:30000/'
     const dbData = {
       links: {
         kibana_url: kibanaURL,
         grafana_url: grafanaURL,
+        swagger_url: swaggerURL,
```
Comment (Aviv-Galmidi) [inline curiefense/ui/src/components/__tests__/SideMenu.spec.ts:26]: There are two tests for Analytics menu items called `should render all Analytics menu items when db key exists` and `should render all Analytics menu items when db key does not exist`

We should alter the Settings menu items test called `should render all Configuration menu items` to match these two tests and check that the swaggerURL is presented correctly from the DB or from defaults

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]
We also need to bump the UI version for any change using `npm version patch` inside the ui folder

File: curiefense/ui/src/components/SideMenu.vue [hunk 2]
```diff
@@ -118,6 +119,7 @@ export default Vue.extend({
   methods: {
     async loadLinksFromDB() {
       const systemDBData = (await RequestsUtils.sendRequest('GET', `db/system/`)).data
+      const swaggerURL = systemDBData?.links?.swagger_url ? systemDBData.links.swagger_url : this.defaultSwaggerURL
       const kibanaURL = systemDBData?.links?.kibana_url ? systemDBData.links.kibana_url : this.defaultKibanaURL
       const grafanaURL = systemDBData?.links?.grafana_url ? systemDBData.links.grafana_url : this.defaultGrafanaURL
       this.menuItems.analytics.kibana = {
```
[the remark this answers is not in the document]
Comment (xavier-rbz) [reply curiefense/ui/src/components/SideMenu.vue:130]: done

File: curiefense/ui/src/components/__tests__/SideMenu.spec.ts (hunk 1 above)
Comment (xavier-rbz) [reply curiefense/ui/src/components/__tests__/SideMenu.spec.ts:26]: done

**Comment by xavier-rbz** [comment]
Thanks for the review @Aviv-Galmidi! I have fixed the mistakes you have identified.

**Review by Aviv-Galmidi — APPROVED** [review]

**Pull request merged**